src/fastgltf.cpp

```diff
@@ -2394,10 +2450,12 @@ std::size_t fg::GltfDataBuffer::getBufferSize() const noexcept {
 
 #pragma region AndroidGltfDataBuffer
 #if defined(__ANDROID__)
-fg::AndroidGltfDataBuffer::AndroidGltfDataBuffer(AAssetManager* assetManager) noexcept : assetManager{assetManager} {}
+fg::AndroidGltfDataBuffer::AndroidGltfDataBuffer(AAssetManager* assetManager) noexcept {
+    androidAssetManager = assetManager;
```

Comment:
I think this is wrong to handle it like this. I could still open a glTF I loaded manually through a GltfDataBuffer and then the global would be nullptr. I think you should add a new global function that sets the global.